src/lib/filterables.ts

```diff
@@ -0,0 +1,207 @@
+interface Filterable {
+	name: string;
+	items: string[];
+}
+
+const filterables: Filterable[] = [
+	{
+		name: 'Barrows',
+		items: [
```

Comment:
these arrays should  be arrays of item IDs, using `transformArrayOfResolvableItems()` util (and some code below changed after)